serializer.rst

```diff
@@ -138,11 +138,53 @@ To use annotations, first add support for them via the SensioFrameworkExtraBundl
     $ composer require sensio/framework-extra-bundle
 
 Next, add the :ref:`@Groups annotations <component-serializer-attributes-groups-annotations>`
-to your class and choose which groups to use when serializing::
+to your class::
+
+    // src/Entity/Product.php
+    namespace App\Entity;
+    
+    use Doctrine\ORM\Mapping as ORM;
+    use Symfony\Component\Serializer\Annotation\Groups;
+    
+    /**
+    * @ORM\Entity()
+    */
+    class Product
+    {
+    
+       /**
+         * @ORM\Id
+         * @ORM\GeneratedValue
+         * @ORM\Column(type="integer")
+         * @Groups({"show_product", "list_product"})
+         */
+        private $id;
+
+        /**
+         * @ORM\Column(type="string", length=255)
+         * @Groups({"show_product", "list_product"}) 
+         */
+        private $name;
+
+        /**
+         * @ORM\Column(type="integer")
+         * @Groups({"show_product", "list_product"}) 
+         */
+        private $price;
+        
+        /**
+         * @ORM\Column(type="integer")
+         * @Groups({"show_product"}) 
+         */
+        private $description;
+
+    }
+
+and choose which groups to use when serializing::
```

Comment:
Just a personal preference; instead of continuing sentences after an example I prefer having a full sentence to make it easier to read without having to catch up with the previous sentence after looking at the example.

```suggestion
You can now choose which groups to use when serializing::
```